book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] nice work so far. please ensure A method for adding rentals should be present. see more [here](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars`)
- please kindly ensure `the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association`
eg 
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```